.github/workflows/notes.md

```diff
@@ -6,8 +6,26 @@ ${NVIM_VERSION}
 
 ### Windows
 
-1. Extract **nvim-win64.zip**
-2. Run `nvim-qt.exe`
+#### Zip
+
+1. Download **nvim-win64.zip**
+2. Extract the zip.
+3. Run `nvim-qt.exe`
+
+#### MSI
+
+1. Download **nvim-win64.msi**
+2. Run the MSI
+3. Add the Neovim location to your path.
+   - Default location is `C:\Program Files\Neovim`
+4. Run `nvim` or `nvim.exe`
```

Comment:
Shouldn't this still generally be nvim-qt that gets run, like in the Zip instructions?

Reply:
See the discussion below for the change suggestion. I'd expect `nvim-qt` from the start menu and `nvim` from the CLI. If you disagree, I'm happy to revert it back "Run `nvim-qt.exe` :)